src/pages-and-resources/live/Settings.jsx

```diff
@@ -20,9 +24,11 @@ const LiveSettings = ({
   intl,
   onClose,
 }) => {
+  const [isZoomBtnClicked, setIsZoomBtnClicked] = useState(false);
```

Comment:
This state is unnecessary. Use redux store state (isZoomGlobalCredSet) for manage this action.

Reply:
currently, redux `isZoomGlobalCredSet` state is not synced with backendAPI.